tests/fiaas_deploy_daemon/crd/test_crd_watcher.py

```diff
@@ -173,6 +173,9 @@ def test_deploy(self, crd_watcher, deploy_queue, spec_factory, watcher, app_spec
 
         app_spec = app_spec._replace(name=app_name, namespace=namespace, deployment_id=deployment_id)
         spec_factory.return_value = app_spec
+        lifecycle_subject = None
```

Comment:
Maybe remove this line?

Reply:
Ah, yeah. Left over from a previous attempt. It's gone with 84ca20f72438c15331725f39ad61039c1df68517